### DISCUSSION TITLE
Tournament client score difference display

### DISCUSSION THREAD

**Discussion by integerrr** [ppy/osu#15723] [Ideas]
The score difference between the 2 teams is often more important than the absolute score of the respective teams

Current:
![image](https://user-images.githubusercontent.com/7279624/142779583-2511442b-fbef-41fa-bf09-ea3d942479c0.jpeg)

Suggestion:
![image](https://user-images.githubusercontent.com/7279624/142779595-4a828a86-4454-4d22-93b4-5e871735130a.jpeg)

**Comment by peppy** [comment]
not sure about this.
